sample/src/main/java/com/canhub/cropper/sample/extend_activity/app/SExtendActivityJava.java

```diff
@@ -0,0 +1,34 @@
+package com.canhub.cropper.sample.extend_activity.app;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.canhub.cropper.CropImage;
+
+public class SExtendActivityJava extends AppCompatActivity {
```

Comment:
Please change the name for `SCameraActivityJava` this is not extending the library `CropImageActivity`